### PR TITLE
Fix for LIG UplinkSets Idempotency

### PR DESCRIPTION
**Pull request by yuvirani**
### Description
Fix for LIG UplinkSets Idempotency

### Check List
- [x] New functionality includes sanity testing.
  - [x] All sanity tests pass. (`$ ansible-test sanity`).
  - [ ] All unit tests pass.
- [x] New functionality has been documented in the README if applicable.
  - [x] New functionality has been thoroughly documented in the examples (please include helpful comments).

### CODE REVIEW

File: plugins/modules/oneview_logical_interconnect_group.py [hunk 1]
```diff
@@ -149,7 +149,7 @@
     type: dict
 '''
 
-from ansible_collections.hpe.oneview.plugins.module_utils.oneview import OneViewModule, OneViewModuleResourceNotFound
+from ansible_collections.hpe.oneview.plugins.module_utils.oneview import OneViewModule, OneViewModuleResourceNotFound, compare_lig, dict_merge, LIGMerger
```
Comment (VenkateshRavula) [inline plugins/modules/oneview_logical_interconnect_group.py:152]: remove dict_merge

File: plugins/module_utils/oneview.py [hunk 2]
```diff
@@ -958,9 +1155,7 @@ def _merge_connections(self, merged_data, resource, data):
         return merged_data
 
     def _merge_connections_boot(self, merged_data, resource):
-        existing_connection_map = {}
-        for x in resource[SPKeys.CONNECTIONS]:
-            existing_connection_map.update({x[SPKeys.ID]: x.copy()})
+        existing_connection_map = {x[SPKeys.ID]: x.copy() for x in resource[SPKeys.CONNECTIONS]}
```
Comment (VenkateshRavula) [inline plugins/module_utils/oneview.py:1158]: revert this change

Comment (yuvirani) [reply plugins/module_utils/oneview.py]: Done

File: plugins/modules/oneview_logical_interconnect_group.py (hunk 1 above)
Comment (yuvirani) [reply plugins/modules/oneview_logical_interconnect_group.py]: Done

**Review by VenkateshRavula — APPROVED** [review]

**Review by AsisBagga — APPROVED** [review]
